### ISSUE TITLE
Guideline 4.1 typo: "by by

### ISSUE THREAD

**Issue by yatil** [w3c/wcag#3143]
[Source](https://www.w3.org/TR/WCAG21/#robust) . Emphasis mine:

> Content must be robust enough that it can be interpreted **by by** a wide variety of user agents, including assistive technologies.

Should be a quick editorial fix as it has been fixed in WCAG 2.2 draft.

**Comment by rachaelbradley** [comment]
Thank you for raising this issue. It has already been corrected in the Editor's Draft: https://w3c.github.io/wcag/guidelines/22/#robust

**Comment by awkawk** [comment]
https://lists.w3.org/Archives/Public/w3c-wai-gl/2018JulSep/0366.html
https://www.w3.org/WAI/WCAG21/errata/ (6th bullet)

**Comment by yatil** [comment]
This has not been fixed in WCAG 2.1. It might be mentioned in some errata, but as a typo this can be fixed and republished no problem. Nobody reads errata anyway. 

Please fix things. Why is this so hard?

**Comment by yatil** [comment]
Apparently I can’t reopen this issue. Shall I open a new one?

**Comment by yatil** [comment]
Leaving the [link to the Process Document](https://www.w3.org/2021/Process-20211102/#revised-rec-editorial), which allows publication of Recommendations for Errata if there are no votes against doing so. Because apparently the Chairs are not aware of this.

https://www.w3.org/2021/Process-20211102/#revised-rec-editorial

**Comment by hidde** [comment]
Please consider fixing this typo in 2.1, this does not seem like something that is hard or controversial, I don't think it would be reasonable to imagine a world where the fixing of this typo would lead to interpretation issues or other problems.

**Comment by awkawk** [comment]
It won't be fixed until WCAG 2.1 is republished. Assuming that happens all of the errata would be integrated. Opening another issue won't help.

**Comment by yatil** [comment]
@awkawk What would help? I mean, this is standard procedure, outlined in the process document. Why is this not addressed? Baffling.

**Comment by yatil** [comment]
Again, what can be done to make the WG Chairs republish WCAG 2.1 for typos, which doesn’t even need Group approval as per the previously shared link.

**Comment by rachaelbradley** [comment]
We are actively working towards republishing WCAG 2.1. There are a number of editorial updates we want to to incorporate.

**Comment by yatil** [comment]
Thanks, @rachaelbradley. I appreciate it!
